### PR TITLE
ci(SPV-1138): add separated linter files and fix warnings.

### PR DESCRIPTION
**Pull request by dorzepowski**
# Pull Request Checklist

- [ ] 📖 I created my PR using provided  : [CODE_STANDARDS](https://github.com/bitcoin-sv/spv-wallet-go-client/blob/main/.github/CODE_STANDARDS.md)
- [ ] 📖 I have read the short Code of Conduct: [CODE_OF_CONDUCT](https://github.com/bitcoin-sv/spv-wallet-go-client/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] 🏠 I tested my changes locally.
- [ ] ✅ I have provided tests for my changes.
- [ ] 📝 I have used conventional commits.
- [ ] 📗 I have updated any related documentation.
- [ ] 💾 PR was issued based on the Github or Jira issue.

### CODE REVIEW

**Review by mgosek-4chain — APPROVED** [review]

File: internal/auth/auth.go [hunk 1]
```diff
@@ -1,3 +1,5 @@
+// Package auth is responsible for handling the authentication of the requests to the server.
+// nolint: unused,nolintlint // FIXME: this file will be used soon
```
Comment (chris-4chain) [inline internal/auth/auth.go:2]:
```suggestion
// nolint: unused,nolintlint // FIXME: this file will be modified soon
```

File: this_file_should_be_removed.go
Comment (chris-4chain) [inline this_file_should_be_removed.go:1]: xD

**Review by chris-4chain — APPROVED** [review]

File: internal/auth/auth.go (hunk 1 above)
Comment (dorzepowski) [reply internal/auth/auth.go:2]: no, it will be used, not modified :D

Comment (chris-4chain) [reply internal/auth/auth.go:2]: yea :D I get it 😄

**Pull request merged**